packages/library-base/src/commonMain/kotlin/io/realm/kotlin/log/LogLevel.kt

```diff
@@ -20,3 +21,31 @@ public enum class LogLevel(public val priority: Int) {
     WTF(6),
     NONE(7);
 }
+
+internal fun LogLevel.toCoreLogLevel(): CoreLogLevel {
```

Comment:
I disagree we could make functions internal to a public file, like we do with `RealmLog` is a public singleton and has internal methods.

We created the internal package to place all the internal classes that had to be made public because of the base/sync split.

Reply:
The reason for the `internal` packages was to split it so that it was obvious what the public API was and that users wouldn't end up browsing internal code unless they intentionally navigated to it from the public API. We haven't been completely strict on this, but I don't see why we should not try to follow it 🤷